docs/tasks/declaring-a-pod-not-ready-for-requests.md

```diff
@@ -0,0 +1,115 @@
+---
+synopsis: "Configure a Pod to start receiving requests only after becoming ready."
+objects:
+- readinessProbe
+- probe
+concept_rankings:
+- concept: pod
+  rank: 50
```

Comment:
rank: 2
(Only 1 and 2 are supported right now, these map to "Core" and "Advanced" at the moment)